terraform/templates/vpc/dynamo_endpoint_policy.json.tpl

```diff
@@ -2,7 +2,7 @@
   "Version": "2012-10-17",
   "Statement": [
     {
-      "Sid": "allow-callers-from-specific-account",
+      "Sid": "allowCallersFromSpecificAccount",
```

Comment:
You've made these camel case rather than Pascal.

Reply:
Got the names mixed up, the majority are camel case https://github.com/search?q=repo%3Anationalarchives%2Fdr2-ingest%20%22Sid%22&type=code so I'll leave them.